builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -1063,3 +1066,73 @@ def loadBuildModules(buildModuleUrl){
     }
   }
 }
+
+def servelessDeploy(env, envBucketKey){
+    echo "which env: $env"
+    echo "which bucket key: $envBucketKey"
+  try {
+    sh "serverless deploy --stage ${env} -v --bucket ${configLoader.AWS.S3[envBucketKey]} --force > output.log"
+    return "success"
+  } catch (ex) {
+    echo "Serverless deployment failed due to $ex"
+  }
+
+  def outputLog = readFile('output.log').trim()
+  echo "serverless deployment log $outputLog"
+  return outputLog
+}
+
+
+def isExistingLogGroup (groupName){
+	def isExistingLogGrp = false
+	def filterLogs = sh (
+		script: "aws logs describe-log-groups --log-group-name-prefix $groupName --output json --profile cloud-api",
+		returnStdout: true
+	).trim()
+	def toJSON = new groovy.json.JsonSlurper()
+	def filterLogsJSON = toJSON.parseText(filterLogs)
+	def logGroups = filterLogsJSON.logGroups
+	echo "logGroups : $logGroups"
+	if(logGroups.size() > 0  && logGroups[0].logGroupName == groupName){
+		isExistingLogGrp = true
+	}
+	return isExistingLogGrp
+}
+
+def removeLogGroup(groupName) {
+	sh "aws logs delete-log-group --log-group-name $groupName --profile cloud-api"
```

Comment:
Use $credsId as the profile name (pass it as function param).

Reply:
Made the changes